meeting-notes/2018/2018-08-20--js-core-dev-team-weekly.md

```diff
@@ -0,0 +1,131 @@
+# ⚡️ⒿⓈ Core Dev Team Weekly Sync 🙌🏽 August 20, 2018
+
+- **Lead:** Alan Shaw (@alanshaw)
+- **Notetaker:** @jacobheun
+- **Attendees:**
+  - @achingbrain
+  - @jacobheun
+  - @vmx
+  - @mgoelzer
+  - _attendee names..._
+- **Recording:** https://youtu.be/KZ5mMAdMmo0
+
+## Agenda
+
+- Ask everyone to put their name into the list of attendees
+- Round of updates
+  - What have you accomplished since the last Weekly?
+  - Were there any blockers? If so, which ones? Is it still blocked? Why?
+  - What is the next important thing you should focus on?
+- Ask for general questions. Could be things like:
+  - I'm stuck with something, I don't know who to ask. Who knows who to ask?
+  - Who can help me with xyz?
+- Plan this week
+  - Select issues to work on
+- Review remaining issues if there is time left
+
+## Notes
+
+### Week Update
+
+@vmx
+  - Done:
+    - (already last week): Describe your DAG in a text file: https://github.com/vmx/dagbuilder
+    - Hello world libp2p (for GraphSync)
```

Comment:
It works? Is there a demo?

Reply:
Check the Q&A below :)

Reply:
It's just a libp2p hello world to get me familiar with the code. So it would be the same as checking out the libp2p examples.